edgelet/edgelet-core/src/lib.rs

```diff
@@ -17,6 +17,7 @@ pub mod crypto;
 mod error;
 mod identity;
 mod module;
+pub mod network;
```

Comment:
Doesn't need to be `pub` since you're already `pub use`ing everything that needs to be exported from it below.